enterprise/server/cmd/executor/BUILD

```diff
@@ -80,6 +81,120 @@ go_binary(
     embed = [":executor_lib"],
 )
 
+container_layer(
+    name = "tini_layer",
+    directory = "/",
+    files = select({
+        "@platforms//cpu:x86_64": [
+            "@com_github_krallin_tini_tini-linux-amd64//file:tini",
+        ],
+        "@platforms//cpu:arm64": [
+            "@com_github_krallin_tini_tini-linux-arm64//file:tini",
+        ],
+        "//conditions:default": [],
+    }),
+    tags = ["manual"],
+)
+
+container_layer(
+    name = "docker_credential_gcr_layer",
+    directory = "/usr/bin",
+    files = select({
+        "@platforms//cpu:x86_64": [
+            "@com_github_googlecloudplatform_docker-credential-gcr-linux-amd64//:docker-credential-gcr.bin",
+        ],
+        "@platforms//cpu:arm64": [
+            "@com_github_googlecloudplatform_docker-credential-gcr-linux-arm64//:docker-credential-gcr.bin",
+        ],
+        "//conditions:default": [],
+    }),
+    tags = ["manual"],
+)
+
+# This is the file that would be written by `docker-credential-gcr configure-docker`
+write_file(
+    name = "docker_credential_gcr_docker_config",
+    out = "config.json",
+    content = [
+        "{",
+        "        \"auths\": {},",
+        "        \"credHelpers\": {",
+        "                \"asia.gcr.io\": \"gcr\",",
+        "                \"eu.gcr.io\": \"gcr\",",
+        "                \"gcr.io\": \"gcr\",",
+        "                \"marketplace.gcr.io\": \"gcr\",",
+        "                \"us.gcr.io\": \"gcr\"",
+        "        }",
+        "}",
+    ],
+)
```

Comment:
nit: this could be simplified by storing the JSON as a plain file under //dockerfiles/executor_image/data/docker/config.json